lib/scrabble.rb

```diff
@@ -1,18 +1,53 @@
 class Scrabble
+  require 'pry'
+  attr_reader :word,
+              :point_values
+  def initialize
+    @word = word
+    @point_values = {
+        "A"=>1, "B"=>3, "C"=>3, "D"=>2,
+        "E"=>1, "F"=>4, "G"=>2, "H"=>4,
+        "I"=>1, "J"=>8, "K"=>5, "L"=>1,
+        "M"=>3, "N"=>1, "O"=>1, "P"=>3,
+        "Q"=>10, "R"=>1, "S"=>1, "T"=>1,
+        "U"=>1, "V"=>4, "W"=>4, "X"=>8,
+        "Y"=>4, "Z"=>10
+      }
+  end
 
   def score(word)
-    1
+    score = []
+    if word == ""
+      return 0
+    elsif word.nil?
+      return 0
+    else
+      upcased = word.upcase
+      split_word = upcased.split(//)
+      split_word.each do |letter|
+        score << @point_values[letter]
+      end
+      return score.inject(:+)
```

Comment:
Since you are using inject/reduce here, take a look at the expanded syntax. You can build it without doing the each step and the inject step.